apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/UrlSigner.V4Signer.cs

```diff
@@ -68,10 +68,15 @@ private struct SigningState
                 private string _resourcePath;
                 private string _canonicalQueryString;
                 internal byte[] _blobToSign;
+                private string _scheme;
+                private string _host;
 
                 internal SigningState(RequestTemplate template, Options options, IBlobSigner blobSigner, IClock clock)
                 {
                     options = options.ToExpiration(clock);
+                    _host = $"{(urlStyle == UrlStyle.VirtualHosted ? $"{bucket}." : string.Empty)}{StorageHost}";
```

Comment:
I think this would actually be simpler without the interpolated string literal:

```csharp
_host = urlStyle == UrlStyle.VirtualHosted ? $"{bucket}.{StorageHost}" : StorageHost;
```

It duplicates the StorageHost part, but avoids nested interpolated string literals and just-for-the-parser parentheses.